src/main/java/com/salesforce/mc/identity/common/retryable/RetryException.java

```diff
@@ -0,0 +1,11 @@
+package com.salesforce.mc.identity.common.retryable;
+
+public class RetryException extends Exception {
+
+	private static final long serialVersionUID = -7407775622713419770L;
```

Comment:
I am assuming this is to avoid InvalidClassException? Is this just a random number?
https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it

Reply:
Its completely irrelevant for this usecase as we are not serializing and deserializing objects, its just to keep my eclipse quite.